Refactor to be more in line with a standard Gradle project structure

gbrodman opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/google/nomulus/90)
<!-- Reviewable:end -->

mindhog commented:
In the build.gradle files, you're adding project.ext to many of the variables.
AIUI, the project.ext prefix should only be necessary when assigning the
value.  Is there a reason to do the qualification?

...
diff --git a/java/google/registry/proxy/resources/hello.xml
b/core/src/main/java/google/registry/ui/epptmpl/hello.xml
similarity index 100%
rename from java/google/registry/proxy/resources/hello.xml
rename to core/src/main/java/google/registry/ui/epptmpl/hello.xml
diff --git a/java/google/registry/ui/epptmpl/hello.xml
b/core/src/test/java/google/registry/flows/session/testdata/hello.xml
similarity index 100%
rename from java/google/registry/ui/epptmpl/hello.xml
rename to core/src/test/java/google/registry/flows/session/testdata/hello.xml
diff --git a/javatests/google/registry/flows/session/testdata/hello.xml
b/core/src/test/java/google/registry/flows/testdata/hello.xml
similarity index 100%
rename from javatests/google/registry/flows/session/testdata/hello.xml
rename to core/src/test/java/google/registry/flows/testdata/hello.xml

What's up with these?

...
diff --git a/javatests/google/registry/flows/testdata/hello.xml
b/proxy/src/main/java/google/registry/proxy/resources/hello.xml
similarity index 100%
rename from javatests/google/registry/flows/testdata/hello.xml
rename to proxy/src/main/java/google/registry/proxy/resources/hello.xml

Is this not a test resource?

gbrodman commented:
1. Ah you're right. Changed the references to be just ${javaDir} 
2. Do you mean the fact that hello.xml files exist at all? They're testing for the HelloFlow I believe, which I think is basically just a ping for EPP commands if I recall correctly 
3. Moved, sorry.
